configure.ac

```diff
@@ -94,12 +94,19 @@ AC_ARG_WITH(abi,
 	)
 
 AC_ARG_WITH(tune,
-	[AS_HELP_STRING([--with-tune=rocket],
-		[Set the base RISC-V CPU, defaults to rocket])],
+	[AS_HELP_STRING([--with-tune=generic],
+		[Set the base RISC-V CPU, defaults to GCC's default])],
 	[],
-	[with_tune=rocket]
+	[]
 	)
 
+if test -n "$with_tune" && test "$with_tune" != "yes"; then
```

Comment:
This is usually done like this: `test "x$with_tune" != "xyes"`.
But here this might be a better fit: `test "x$with_tune" != "x"`.